cli/core/hack/rpm/tanzu-cli.spec

```diff
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
+License:    Apache 2.0
+URL:        https://github.com/vmware-tanzu/tanzu-cli/
+Vendor:     VMware
+Summary:    The Tanzu CLI
+
+%ifarch amd64
+%define arch amd64
+%endif
+
+%ifarch aarch64
+# TODO For now, we use the amd64 build for arm64
+%define arch amd64
+%endif
+
+%undefine _disable_source_fetch
+Source0:    https://github.com/vmware-tanzu/tanzu-framework/releases/download/v%{version}/tanzu-cli-linux-%{arch}.tar.gz
+
+%description
+VMware Tanzu is a modular, cloud native application platform that enables vital DevSecOps outcomes
+in a multi-cloud world.  The Tanzu CLI allows you to control VMware Tanzu from the command-line.
+
+# Go does not generate a build-id compatible with RPM, so we disable the need for a build-id
+# See https://github.com/rpm-software-management/rpm/issues/367
+%global _missing_build_ids_terminate_build 0
+
+# This is required to avoid some missing debug file errors
+%define debug_package %nil
+
+%prep
+%setup -T -c
+
+%build
+# Nothing to build
+
+%install
+rm -rf $RPM_BUILD_ROOT
+mkdir -p $RPM_BUILD_ROOT/%{_bindir}
+# Unpack the CLI and rename it
+tar -xzf %{SOURCE0} --strip-components 1 v%{version}/tanzu-core-linux_%{arch}
+mv tanzu-core-linux_%{arch} $RPM_BUILD_ROOT/%{_bindir}/tanzu
+
+%files
+%{_bindir}/tanzu
```

Comment:
adding changelog section is also recommended to history - the actual changelogs may use a couple of lines of highlights or even point to github changelogs

Reply:
You are right that it would be better, however, it makes automation more difficult at this stage.  As the PR stands now, building a package for a new version does not require any changes to the SPEC file since the `Version` is injected.  Adding %changelog would require manual changes to the SPEC when building the packages.

Once we integrate this package generation with Github actions and have a more complete automation, it may be possible to automate the changelog somehow (although the text will have to be taken from somewhere...).

What do you say to leaving the %changelog out for this MVP and revisiting in the near future when I work on further automation?

Reply:
Correct, the changed SPEC file would need to be committed back. Usually an update to SPEC files is done part of the release rituals, but can be revisited at subsequent iterations 👍